# HG changeset patch
# User Rik <octave@nomad.inbox5.com>
# Date 1310852441 25200
#      Sat Jul 16 14:40:41 2011 -0700
# Node ID 0e65376cc44b392265e0f1a4513b38fbf5ffcec8
# Parent  a12569d5906edafd5582164046ef45cf32bf1947
codesprint: Add 3 tests for unimplemented.m

* unimplemented.m: Add 3 tests.

diff --git a/scripts/help/unimplemented.m b/scripts/help/unimplemented.m
--- a/scripts/help/unimplemented.m
+++ b/scripts/help/unimplemented.m
@@ -421,8 +421,19 @@ function list = missing_functions ()
   "xlsread",
   "xlswrite",
   "xmlread",
   "xmlwrite",
   "xslt",
   "zoom",
   };
 endfunction
+
+
+%!test
+%! str = unimplemented ("no_name_function");
+%! assert (isempty (str));
+%! str = unimplemented ("quad2d");
+%! assert (str(1:51), "quad2d is not implemented.  Consider using dblquad.");
+%! str = unimplemented ("MException");
+%! assert (str(1:58), "the `MException' function is not yet implemented in Octave");
+
+
